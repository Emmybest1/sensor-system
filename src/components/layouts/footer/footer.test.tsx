import { render } from '@testing-library/react';

import { Footer } from '..';

describe('<Footer/>', () => {
  test('should render Component', () => {
    const { container } = render(<Footer />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
