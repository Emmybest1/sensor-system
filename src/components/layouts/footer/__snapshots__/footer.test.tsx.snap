// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer/> should render Component 1`] = `
<footer
  class="sc-gsTEea bHaEwb"
>
  Powered by Emmanuel OnahÂ 
  <time>
    2021
  </time>
</footer>
`;
